README.md

```diff
@@ -264,5 +264,36 @@ Golang agent supports the following dynamic configurations.
 |:-----------------:|:----------------------------------------------------------------------------------------:|:--------------------:|
 | agent.sample_rate | The percentage of trace when sampling. It's `[0, 1]`, Same with `WithSampler` parameter. |         0.1          |
 
+## Process
```

Comment:
```suggestion
## Process Status Hook
```

Process is a common and wide used concept in Linux, I don't want to hijack it as a feature name.

Reply:
Updated.